### ISSUE TITLE
Excessive white space on homepage

### ISSUE THREAD

**Issue by tanyapowell** [womenhackfornonprofits/whfnp#1]
Between the newsletter signup and partners logo there's alot of whitespace. 

![image](https://cloud.githubusercontent.com/assets/11095605/12792711/0f35cf4e-caa5-11e5-94c5-b85c1a0229ea.png)

There's alot of white space around the partner logos. Making the logos bigger would help by reducing the white space on the left and right of the logos

![image](https://cloud.githubusercontent.com/assets/11095605/12792718/1845debc-caa5-11e5-9939-2986fe8961d2.png)

**Comment by lili2311** [comment]
@tanyapowell 
1. Amended the Partner logos to have less space around them, better?
![Do you still an issue regarding the whitespace?](https://cloud.githubusercontent.com/assets/2911613/12889852/cc82c22e-ce78-11e5-83a6-974f3c728d11.png)
2. Do you still an issue regarding the whitespace?
![Amended the Partner logos to have less space around them, better?](https://cloud.githubusercontent.com/assets/2911613/12889853/cc834794-ce78-11e5-90f7-d5a8ce580d26.png)

**Comment by tanyapowell** [comment]
1. :+1: for the Partners
2. Cleared cache and deleted browser history but still getting the extra white space issue on Chrome. Looks fine on Safari though. Worth asking others to check too

**Comment by lili2311** [comment]
https://github.com/womenhackfornonprofits/whfnp/pull/7

**Comment by lili2311** [comment]
Will check all browsers and re-open a new issue once cross browser testing is done
